whisper/test.py

```diff
@@ -29,6 +29,7 @@ def forward_mlx(model, mels, tokens):
     mels = mx.array(mels.transpose(0, 2, 1))
     tokens = mx.array(tokens, mx.int32)
     logits = model(mels, tokens)
+    # todo: fix this
```

Comment:
What's up with this comment? Is the test broken?

Reply:
yeah, since `Whisper`'s `__call__` function now returns all (`logits`, `kv_cache`, `cross_qk`) instead of just `logits`. This inclusion of `cross_qk` is for computing alignments, but we could create another function for this

Reply:
Cool, yea that's probably the simplest fix. The call function should just returns the logits (as is) and another function for the attention scores. But I don't feel strongly about it either way.